manifests/metering-config/expose-route.yaml

```diff
@@ -33,6 +33,11 @@ spec:
         # change REPLACEME to the output of your command
         cookieSeed: "REPLACEME"
 
-        # enables authenticating using a serviceAccount token
+        # Enables authenticating using a bearer token from a serviceAccount or
+        # for your user.
```

Comment:
`serviceAccount or user` to eliminate second person perspective.

Reply:
Done